sys-utils/renice.1.adoc

```diff
@@ -46,16 +46,22 @@ renice - alter priority of running processes
 
 == SYNOPSIS
 
-*renice* [*-n*] _priority_ [*-g*|*-p*|*-u*] _identifier_...
+*renice* [*-n|--priority|--relative*] _priority_ [*-g*|*-p*|*-u*] _identifier_...
 
 == DESCRIPTION
 
 *renice* alters the scheduling priority of one or more running processes. The first argument is the _priority_ value to be used. The other arguments are interpreted as process IDs (by default), process group IDs, user IDs, or user names. *renice*'ing a process group causes all processes in the process group to have their scheduling priority altered. *renice*'ing a user causes all processes owned by the user to have their scheduling priority altered.
 
 == OPTIONS
 
-*-n*, *--priority* _priority_::
-Specify the scheduling _priority_ to be used for the process, process group, or user. Use of the option *-n* or *--priority* is optional, but when used it must be the first argument.
+*-n* _priority_::
+Specify the *absolute* scheduling _priority_ to be used for the process, process group, or user. Use of the option *-n* is optional, but when used it must be the first argument. *Note* that the argument is *absolute* for historical reasons. To make this flag 100% posix compliant, define POSIXLY_CORRECT in the environement and it will use a *relative* value instead.
+
+*--priority* _priority_::
+Specify the *absolute* scheduling _priority_ to be used for the process, process group, or user. Use of the option *--priority* is optional, but when used it must be the first argument.
```

Comment:
The descriptions for `-n`, `--priority`, `--relative` are very repetitive.

Reply:
I'll rework them, no problem.